### ISSUE TITLE
Screen Reader: Many Pages: Various links are announced without a role

### ISSUE THREAD

**Issue by olenamihno** [Expensify/App#77567]
## Action Performed:
Precondition: The User should be logged in

1. Using iOS VoiceOver or Android TalkBack, open the Expensify app
2. Swipe to the 'Account' button at the bottom right corner of the screen and double-tap to activate
3. Swipe between various links (Help, What's new)
4. Note how the control is announced

Other Occurrences:
1. On About
2. On About - App download links

## Expected Result:
The controls that open the external browser should be announced with a link role.

## Actual Result:
Various links are announced without a role, e.g.: Help and What's New

## Area issue was found in:
Profile

## Failed WCAG checkpoints
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User impact:
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested resolution:
Apply accessibilityRole="link" to the controls that open the external browser.
https://reactnative.dev/docs/touchablewithoutfeedback#accessibilityrole

## Environment:
Samsung Galaxy S24 Ultra - Android 14, Partner - Native (No mobile browser)
Apple iPhone 11 - iOS 17.5.1, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos

https://github.com/user-attachments/assets/1d7af307-7691-4689-9b0e-99d51b119351

<img width="828" height="1792" alt="Image" src="https://github.com/user-attachments/assets/d6630d28-aa3a-4e3e-8f7d-a2daa7f6bbdd" />

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** No
**Logs:** https://stackoverflow.com/c/expensify/questions/4856  NA
**Notes/Photos/Videos:** Any additional supporting documentation  Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7006033
**Issue reported by:** NA
**Slack conversation:** NA
